src/stores/messages/MessagesStore.ts

```diff
@@ -264,6 +270,7 @@ export default class MessagesStore {
 		this.selectedMessageId = null;
 		this.highlightedMessageId = null;
 		this.hintMessages = [];
+		this.exportStore.disableExport();
```

Comment:
we can delete this line since reaction to `filterStore.filter` will be triggered